### ISSUE TITLE
Error while loading datasets in True inference with LayoutLMv2ForTokenClassification + Gradio demo.ipynb

### ISSUE THREAD

**Issue by akashlp27** [NielsRogge/Transformers-Tutorials#157]
OSError: Cannot find data file. 
Original error:
cannot identify image file '/root/.cache/huggingface/datasets/downloads/extracted/e5bbbc543f8cc95554da124f3e80a57ed24d67d06ae1467da5810703f851e3f9/dataset/training_data/images/0000971160.png'

while loading dataset from
```
from datasets import load_dataset 
datasets = load_dataset("nielsr/funsd")
```

**Comment by NielsRogge** [comment]
cc'ing @mariosasko. Also having this issue.

Got a similar issue with "nielsr/funsd-image-feature", even though this worked fine in the past.

**Comment by mariosasko** [comment]
Hi! These datasets work without any issues for me locally or in Colab. Can you please paste the entire stack trace and the output of the `datasets-cli env` command?

**Comment by NielsRogge** [comment]
Yes I'm currently also not able to reproduce it. It's a weird issue, seems flaky.

If I encounter it again, will report here.
